### DISCUSSION TITLE
Get back to surf when closing a game

### DISCUSSION THREAD

**Discussion by msx80** [nesbox/TIC-80#2575] [Q&A] [answered]
Hi, i'm probably missing something obvious, but when i close a game that i started with surf, i get back to the console. Is there a way to get back to surf instead? We are going to run tic80 on an event and we would like it to be in a kind of "kiosk mode", running games off of surf.

I've found that B button starts surf again from console but it's not the best solution.

**Comment by Skeptim** [comment, marked as answer]
I don't think that's on purpose but if you set the `dev mode` to `on` in the menu, then Esc will go back to surf instead of the console.

**Reply by msx80** [reply]
well that was fast, thanks! skipping the menu is even better for me, problem solved
